### DISCUSSION TITLE
MongoDB "In" Query

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#9091] [Help] [answered]
Hi,

So I think this may be a bug but I'll start here.

I'm trying to return a few documents from a collection using the "in" operator.

{"_id":{"$in":["ObjectId('639f3e8d46447d7af2d312f4')"]}}

I've tried several variations of the statement above. I am able to use the "in" operator with other fields. I'm just unable to use the in perator for the ID. Is this a bug or just something I'm doing wrong?

Best,

J

**Comment by melohagan** [comment, marked as answer]
This does appear to be a bug. 

Thanks for raising the issue: https://github.com/Budibase/budibase/issues/9119
